README.md

```diff
@@ -10,14 +10,123 @@
 
-Should explain the rails console.
+In the same way, when it comes to building a web application it would technically be possible to build out all of the functionalities needed yourself, but it's typically a better idea to leverage a system that has already spent over a decade developing the tools necessary for getting applications built.
+
+
+## What is Ruby on Rails?
+
+* **A web framework** - A web framework provides developers the tools they need in order to build applications. While every application is unique there are certain components that can be found in almost every application, such as: routing, asset management, database connections, and the list goes on. A good web framework gives developers these baseline tools so that they don't have to create the base application functionality for each new project.
+* **A Ruby Gem** - At its core, Ruby on Rails is simply a set of Ruby code libraries, and since the entire codebase is open source you have the ability to review the framework to better understand how it works.
+* **A MVC framework** - MVC stands for Model-View-Controller, this essentially means that Rails takes advantage of the popular application architecture that helps developers naturally separate concerns and organize their applications properly. This setup encourages a specific set of conventions, such as placing the logic for the application in the model files, the code flow in the controllers, and the views to simply display content to the user.
+
+## What Ruby on Rails is not
+* **A programming language** - This is one of the most common misconceptions, Ruby on Rails is not a programming language, instead it is a set of code libraries built in Ruby.
+* **A slow framework** - Due to the fact that Rails is one of the most straightforward frameworks to learn, it can lead to a number of poor coding practices from beginners. However if built properly, Rails projects can be as fast as any other framework. Furthermore, with Rails service based architecture it makes it a perfect candidate for microservice applications, which can be some of the fastest and best performing applications on the web.
+
+# Creating your first Rails project
+
+*Before you continue, this assumes that you have Ruby, RubyGems, and Bundler installed on your system*
+
+## Installing the Rails Gem
+
+As mentioned above, Rails is simply a Ruby Gem, to install it on your system, run the following command in the terminal of your computer:
+
+```gem install rails```
+
+Depending on your system configuration you may need to prepend the command with ```sudo``` to install the gem as the root user. Once the gem is installed you can create Rails applications!
```

Comment:
Before diving in, explain over the rest of this content we'll go over everything in this directory so don't worry if you are overwhelemed!